### PR TITLE
Front+back

### PR DESCRIPTION
**Pull request by jibinjEcobee**
We have the backend working, and the front end. It is currently configured for Jira and Testrail, but only for AdminPortal

here is a 📸 of the app
screen shot 2018-12-11 at 3 13 14 pm

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (JohnZecobee) [inline .vscode/launch.json]: Please remove and ignore this file

File: frontend/src/main.js [hunk 2]
```diff
@@ -1,36 +1,31 @@
...
+import { createHttpLink } from "apollo-link-http";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import VueApollo from "vue-apollo";
+import App from "./App.vue";
+import Vuetify from "vuetify";
+import 'vuetify/dist/vuetify.min.css' 
+
 const link = createHttpLink({
-  uri: 'http://10.90.5.101:4000',
+  uri: "http://localhost:5000"
```
Comment (JohnZecobee) [inline frontend/src/main.js]: We need to switch back to the IP address for master @psean-eco

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+const server = new GraphQLServer({
+  typeDefs: "./src/schema.graphql",
+  resolvers
+});
+const option = {
+  port: 5000
+};
+server.start(option, () =>
+  console.log(`Server is running on localhost:${option.port}`)
```
Comment (JohnZecobee) [inline src/index.js]: is this accurate? @psean-eco

File: src/resolvers/Project.resolvers.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    return result;
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+const automationResults = async ({ name }) => {
+  try {
+    const response = await axios.get(
+      `http://jenkinsqa.ecobee.com/jenkins/job/Admin_Portal_v2.10.0_Dev/lastCompletedBuild/api/json`
```
Comment (JohnZecobee) [inline src/resolvers/Project.resolvers.js]: Can we build this URL?  I now have a job that is called `Admin Portal` so we can drop Dev and the Release number

Comment (psean-eco) [reply src/resolvers/Project.resolvers.js]: yeah this needs to be updated

File: .vscode/launch.json (hunk 1 above)
Comment (jibinjEcobee) [reply .vscode/launch.json]: done

**Pull request merged**